.github/workflows/CI.yml

```diff
@@ -32,7 +32,6 @@ jobs:
   check_versions:
     name: Check Versions
     runs-on: ubuntu-latest
-    needs: check_changelog
```

Comment:
Why was this changed? It's required here.

Check changelog runs first (doesn't run except on PRs or Pulls into master branch). Makes sure that the version in the changelog was updated and is not equal to the latest released version.

Then check versions runs... makes sure that the version number in the CMakeLists.txt was updated.